### ISSUE TITLE
Failure when trying to use NerdsAndCompany\Schematic

### ISSUE THREAD

**Issue by nickroberts** [spicywebau/craft-neo#286]
I installed the [Craft webhooks plugin](https://github.com/craftcms/webhooks), and when creating a new webhook (or editing an existing one), `An unknown error occurred.` shows up.

I haven't seen this before, until using the Webhooks plugin, and looks to be related to getting an auto complete list for classes.

URL: `/index.php?p=admin/actions/webhooks/webhooks/class-suggestions`
```
PHP Fatal Error 'yii\base\ErrorException' with message 'Class 'NerdsAndCompany\Schematic\Converters\Base\Field' not found' 

in /var/www/html/vendor/spicyweb/craft-neo/src/converters/Field.php:14

Stack trace:
#0 [internal function]: yii\base\ErrorHandler->handleFatalError()
#1 {main}
```

It doesn't seem to affect anything, aside from not allowing the classes to autocomplete.

Thanks, again, for the great work on this plugin. We love it!

**Comment by pvldigital** [comment]
Hi @nickroberts,

To clarify, you have both the webhooks and schematic plugin installed and active correct?

**Comment by nickroberts** [comment]
I only have the webhooks plugin installed, not the schematics plugin.

**Comment by pvldigital** [comment]
Gotcha. I was able to replicate the issue so i'll work on a fix for it.

**Comment by pvldigital** [comment]
update on this issue:

I don't think this is inherently an issue with neo. Weirdly enough disabling + reenabling neo fixes the issue. I assume it has something to do with the loading of the webhook plugin.

**Comment by nickroberts** [comment]
Well that's weird! I disabled, then re-enabled the plugin and everything seems ok. Thanks!

**Comment by nickroberts** [comment]
It came back (I think after saving the Webhook). Any other ideas?

**Comment by pvldigital** [comment]
Oh really? Hmm I’ll need to look into this further.

**Comment by pvldigital** [comment]
Hi @nickroberts,

Fix has been implemented in release 2.5.10.
